src/tmt_web/templates/status_callback.html.j2

```diff
@@ -0,0 +1,40 @@
+{% extends '_base.html.j2' %}
+{% set title = "Task Status" %}
+
+{% block content %}
+  {% if status == "PENDING" %}
+    <h1>Processing...</h1>
+    <p>Status: {{ status }}</p>
+    <p>Your request is being processed. Please try again in a few seconds:</p>
+    <p><a href="{{ status_callback_url }}">{{ status_callback_url }}</a></p>
+    <script>
+      setTimeout(function() {
+        window.location.reload();
```

Comment:
I forgot to fix this(or just remove it) it was intended as some sort of autorefresh, but instead it just creates a new task. Will get to it asap.

Reply:
fixed during rebasing to main